app/assets/stylesheets/admin.scss

```diff
@@ -32,9 +31,34 @@
   }
 }
 
+// Weekly stats
+.week-selector {
+  margin-top: 18px;
+}
+
+// Pod stats
+.segment-selection {
+  select,
+  [type=submit] {
+    margin-top: 10px;
```

Comment:
It's not. I made `.week-selector` `margin-top: 10px` too. Note that I didn't spend much time building a nice design, there are still inconsistencies, some titles are `h1` when others are `h2` for example. The markup can be refactored to something cleaner but those are admin pages and I didn't want to spend too much time rewriting them. The focus here is to make them available on mobile with a usable interface.